src/List/Filters.component.js

```diff
@@ -119,8 +119,7 @@ export default class Filters extends React.Component {
 
         const inFilter = field => (_(field).isEmpty() ? null : ["in", field]);
 
-        return {
-            query: searchString,
+        let returnObj = {
```

Comment:
The typical approach: use `let` for variables which are assigned more than once (example: a let + conditional with sets it on the different branches). For all other cases: `const`.